src/linear_algebra/finite_dimensional.lean

```diff
@@ -793,6 +793,27 @@ section top
 theorem findim_top : findim K (⊤ : submodule K V) = findim K V :=
 by { unfold findim, simp [dim_top] }
 
+lemma algebra_hom.bijective {F : Type*} [field F] {E : Type*} [field E] [algebra F E]
```

Comment:
Please make these changes in a separate PR.

Reply:
#4793